modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/enums/liferayPicklistName.ts

```diff
@@ -18,4 +18,10 @@ export enum LiferayPicklistName {
 	REGIONS = 'Regions',
 	TARGET_AUDIENCE_ROLES = 'Target Audience/Roles',
 	TARGET_MARKETS = 'Target Markets',
+	PROJECT_INFORMATION = 'Project Information',
```

Comment:
```suggestion
	PROJECT_INFORMATION = 'Project Informations',
```
o definition precisa estar no plural.